### DISCUSSION TITLE
Crash report. Exception: Could not load type 'Barotrauma.AlienRuinMission'

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14244] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

game crashes just after round begins
crash report:

[crashreport (9).log](https://github.com/user-attachments/files/16048209/crashreport.9.log)

save file:
[20240701.zip](https://github.com/user-attachments/files/16048207/20240701.zip)

### Reproduction steps

1. Unpack and place to Barotrauma folder config_player.xml and serversettings.xml 
2. Unpack and place save file to %AppData%\Local\Daedalic Entertainment GmbH\Barotrauma" 
3. Load save file
4. Open Nav terminal

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Exception is from [Fewer Sonar Markers](https://steamcommunity.com/sharedfiles/filedetails/?id=3198868074), and i thought i already fixed that:
![Screenshot_3](https://github.com/FakeFishGames/Barotrauma/assets/122838333/b309ce3c-8b5a-4397-85e3-d81f5b56b050)
Are you sure your mod is updated?

Also attached save is probably wrong, there's no AlienRuinMission there

**Comment by pavel4win** [comment]
AlienRuinMission absent but crash present. Try to load the save with all mods from config_player.xml installed

**Comment by Regalis11** [comment]
Seems to be an issue caused by a mod, closing
